### ISSUE TITLE
research.swtch.com/feed.atom is not readable by FreshRSS

### ISSUE THREAD

**Issue by parkr** [rsc/swtch#5]
Hello @rsc,

TL;DR: I believe your blog's feed is being served with the incorrect `Content-Type`. It should be [`application/atom+xml` instead of `text/plain`](https://validator.w3.org/feed/docs/warning/UnexpectedContentType.html).

I have followed your blog's feed with Feedly for a while. However, since switching to FreshRSS, I am unable to pull your blog's feed at https://research.swtch.com/feed.atom:

```
A feed could not be found at `https://research.swtch.com/feed.atom`; 
the status code is `200` and content-type is `text/plain; charset=utf-8` 
[https://research.swtch.com/feed.atom]
```

I took a look at two validators, both of which ([1][1], [2][2]) indicated that your feed was served with `text/plain`. [FreshRSS uses a library called SimplePie which requires that the content be served with a Content-Type indicating it's an XML file](https://github.com/FreshRSS/FreshRSS/blob/fd7157e40ca32fb1bcda51f1aaf484cbb7988bdd/lib/SimplePie/SimplePie/Locator.php#L162-L164).

Would you be able/willing to serve the /feed.atom file from your blog with a different Content-Type, like `application/atom+xml`, which would be supported by a wider range of RSS readers? Thanks! 

[1]: https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fresearch.swtch.com%2Ffeed.atom
[2]: https://simplepie.org/demo/?feed=https%3A%2F%2Fresearch.swtch.com%2Ffeed.atom
